tests/basic/pool_function_test.go

```diff
@@ -143,6 +144,72 @@ var _ = Describe("{PoolExpandSmoky}", func() {
 		dash.VerifyFatal(resizeErr, nil, "Pool expansion does not result in error")
 		verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
 	})
+
+})
+
+var _ = Describe("{PoolExpandRejectConcurrent}", func() {
+	BeforeEach(func() {
+		contexts = scheduleApps()
+	})
+
```

Comment:
Why are we not initializing a testRailID and not reporting to Aetos?

Reply:
Added!